alembic/versions/9f8ce88d37e9_create_requests_table.py

```diff
@@ -0,0 +1,35 @@
+"""create requests table
+
+Revision ID: 9f8ce88d37e9
+Revises: 
+Create Date: 2019-12-05 16:41:19.128010
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '9f8ce88d37e9'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'requests',
+        sa.Column('id', sa.Integer, primary_key=True),
+        sa.Column('created_at', sa.DateTime, server_default=sa.func.current_timestamp()),
+        sa.Column('prediction_duration', sa.String),
+        sa.Column('edition_id', sa.Integer),
+        sa.Column('content', sa.Text),
```

Comment:
Might this contain draft text which is sensitive? i.e. budgets to be announced? If so, should we be storing this?

Reply:
I have also been looking around to determine how SQLAlchemy maps it's `Text` type to Postgres and whether there it specifies a max length on the column it creates which might impact us, depending how much content is stored. From what I can tell it doesn't add a max length, which seems to be supported by https://github.com/sqlalchemy/sqlalchemy/issues/4443. 1 GB looks to be the maximum column size supported by Postgres for text types - https://www.postgresql.org/docs/current/datatype-character.html.

Reply:
As discussed